trio/_highlevel_generic.py

```diff
@@ -72,34 +77,34 @@ class StapledStream(HalfCloseableStream, metaclass=Final):
 
     """
 
-    send_stream = attr.ib()
-    receive_stream = attr.ib()
+    send_stream: SendStream = attr.ib()
+    receive_stream: ReceiveStream = attr.ib()
 
-    async def send_all(self, data):
+    async def send_all(self, data: bytes | bytearray | memoryview) -> None:
```

Comment:
docstring of `SendStream.send_all` says `bytes|bytearray|memoryview`. `trio-typing` says `bytes|memoryview`.